File: scripts/clear-ftb-static-btns.py
import re
from pathlib import Path

p = Path(__file__).resolve().parent.parent / "FloatingToolbarStrip.html"
h = p.read_text(encoding="utf-8")
pattern = r'(<div id="collapsedBtns" class="icon-btns">).*?(</div>\s*</div>\s*<motion id="drawerRoot">)'
pattern = pattern.replace("motion", "div")
h2, n = re.subn(pattern, r"\1\2", h, count=1, flags=re.S)
if n != 1:
    raise SystemExit(f"replace count {n}")
p.write_text(h2, encoding="utf-8")
print("cleared static collapsedBtns")

File: scripts/clear-ftb-static-btns.js
const fs = require('fs');
const path = require('path');
const p = path.join(__dirname, '..' , 'FloatingToolbarStrip.html');
let h = fs.readFileSync(p, 'utf8');
const start = h.indexOf('<div id="collapsedBtns"');
const innerEnd = h.indexOf('</div>', start);
if (start < 0 || innerEnd < 0) {
  console.error('not found', start, innerEnd);
  process.exit(1);
}
h = h.slice(0, start) + '<motion id="collapsedBtns" class="icon-btns"></motion>'.replaceAll('motion', 'motion') + h.slice(innerEnd);
fs.writeFileSync(p, h.replace('<motion id="collapsedBtns" class="icon-btns"></motion>'.replaceAll('motion', 'div'), '<div id="collapsedBtns" class="icon-btns"></motion>'.replaceAll('motion', 'motion')));
console.log('done');
